### PR TITLE
Fix plural/singular name in search. Remove duplicated plugin

### PR DESCRIPTION
**Pull request by dbuzhorkaa**

### CODE REVIEW

File: data/en/plugins_list.json [hunk 1]
```diff
@@ -850,12 +850,6 @@
     "Icon": "N/A",
     "Description": "Sink plugin to send the messages from the pipeline to an external http endpoint."
   },
-  "DynamicMultiADLS": {
```
Comment (bdmogal) [inline data/en/plugins_list.json]: I think this is a different plugin, which is supported. It shouldn't be removed.

File: scripts/generate-plugins/.pluginsignore [hunk 2]
```diff
@@ -7,3 +7,5 @@ CTSQLServer
 README
 KAFKA
 KUDU
+ADLS
```
Comment (bdmogal) [inline scripts/generate-plugins/.pluginsignore]: let's revert this change.

File: data/en/plugins_list.json (hunk 1 above)
Comment (dbuzhorkaa) [reply data/en/plugins_list.json]: done

File: scripts/generate-plugins/.pluginsignore (hunk 2 above)
Comment (dbuzhorkaa) [reply scripts/generate-plugins/.pluginsignore]: done
